scripts/pi-hole/js/queries.js

```diff
@@ -515,12 +515,17 @@ function tooltipText(index, text) {
 }
 
 function addColumnFilter(event, colID, filterstring) {
-  // Don't do anything when NOT explicitly requesting multi-selection functions
-  if (!event.ctrlKey && !event.metaKey && !event.shiftKey) {
+  if (event.ctrlKey || event.metaKey) {
+    if (colID === 2) {
```

Comment:
Do we want to limit this to the `domain` column only? I wonder how long it will take until users want it for the `client` column, etc.

Reply:
Yeah... I had this thought, too. The conclusion I came to was that there is not really any value in having the value of `Type`, `Status` or `Reply` in the clipboard. But we can expand it to `Client`, perhaps?

Reply:
I have now included the Client column. Unless you can convince me otherwise, I don't see any reason to include the other columns!